detection-rules/impersonation_ledger.yml

```diff
@@ -7,21 +7,18 @@ type: "rule"
 source: |-
   type.inbound
   and (
-    (
       sender.email.domain.root_domain == 'ledger.com'
       and headers.return_path.domain.root_domain not in ('ledger.com', 'amazonses.com')
-    ) or (
-      ilike(sender.email.email, '*ledger.com*')
-      and sender.email.domain.root_domain != 'ledger.com'
-    ) or (
+  )
+  or (
```

Comment:
is this `or` at the right level?
it means that everything below here doesn't need to match `type.inbound`

Reply:
you are absolutely right, will update